docs/intro.md

```diff
@@ -4,4 +4,16 @@ sidebar_position: 1
 
 # Intro
 
-A hacker's guide to hackathons, made by the HackUTD Team.
+Most hackathon projects will want some sort of user interface - after all, you want the judges to be able to visualize your idea, right? Many projects also contain some sort of system that deals with data, whether that be user data or data from external sources. This guide explains and provides resource to learn about user interfaces and data driven systems.
+
+## The User Interface (Frontend)
+
+User interfaces and the software behind them are also known as the **frontend**. Frontends are what users see and interact with. When you go to a website and log in, you're interacting with the **frontend** of that website. The term **frontend** describes the text, buttons, links, input fields, etc. on a webpage and the software that goes into creating and composing them.
+
+<Docs items={[{type: 'link', href: '/frontend', label: 'Frontend'}]}/>
+
+# Data Driven Systems (Backend)
+
+In most hackathon projects, this comes in the form of a **backend**. Imagine you are building the Google search website. You have the search bar, buttons, etc., but how do we actually get the search results? This is where a **backend** comes in handy. **Backends** usually process data and perform calculations on a remote computer (server). The **backend** may be a server or many servers that send and recieve data from a database, run expensive algorithms, control user security and login, or all of these.
+
```

Comment:
Same thing, you can also say technologies like MongoDB Firebase Spring. Just so the reader can make more connections to what technologies are associated with this.